process/sync/baseSync.go

```diff
@@ -227,9 +237,13 @@ func (boot *baseBootstrap) loadNotarizedBlocks(blockFinality uint64,
 			return process.ErrBoostrapFromStorage
 		}
 
-		for i := highestNonceInStorer - blockFinality - lastBlocksToSkip; i <= highestNonceInStorer-lastBlocksToSkip; i++ {
+		firstNonceToApplyFromStorer := highestNonceInStorer - blockFinality - lastBlocksToSkip
```

Comment:
looks very similar to the loadBlocks method. Can you merge them some how?

Reply:
Refactored, and extract the common code